src/components/InputField/PasswordField.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
+import Image from 'next/image'
+
+interface PasswordField {
+  readonly name: string
+  readonly label: string
+  readonly placeholder: string
+  readonly errorMessage: string
+}
+
+const PasswordField = ({ label, placeholder, errorMessage }: PasswordField) => {
+  const [isPasswordShown, setPasswordShown] = useState(false)
+  const placeholderDisplay: string = errorMessage && placeholder
+  const inputType: string = isPasswordShown ? 'text' : 'password'
+
+  const imageSelected: string = isPasswordShown
+    ? '/icons/visible-eye-icon.svg'
+    : '/icons/invisible-eye-icon.svg'
+
+  const inputClassName: string = errorMessage
+    ? 'form__input'
+    : 'form__input form__input--invalid'
+
+
+  const helpClassName: string = errorMessage
+    ? 'form__help'
+    : 'form__help form__help--visible'
```

Comment:
These variables could be moved inside its `render` function since they are not being used anywhere else